_posts/EternalFootman/2016-04-29-final-project.md

```diff
@@ -0,0 +1,49 @@
+
+	Another attitude aspect that I’ve had trouble with is that I’m dreadful at doing something with no purpose. This came up in this game with the dictionary and especially with the external data file. These, as I’m sure you can tell, add nothing to the game; in fact, they make the code more complicated than it needs to be. I struggled with them in part because they were the least familiar concepts (so I can see the point of requiring their inclusion – I do understand them a little better now) but also in part because they felt like a waste of my time. It is part of the learning process, I suppose, but I look forward to doing projects that use things with a purpose.
+
+	This project ended up being an excellent conclusion to the course for me. The concept allowed me to go back over everything I had learned and rework it to the best of my abilities, showcasing how much those have improved. I have learned how to use milestones reasonably effectively (compare with the ones from the drawing app to see the full learning curve). The theme gave it focus, and my love for the source material kept me passionate about making it as good as possible. It’s a peculiar, idiosyncratic little game, and I’m not at all sure that it will make sense to anyone else, but I rather love it.
```

Comment:
peculiar, idiosyncratic games are sometimes the best!  especially if they inpire you to do more with code.